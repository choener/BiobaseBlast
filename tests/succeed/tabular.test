# BLASTP 2.4.0+
# Query: UniRef50_A0A009DWD5 Uncharacterized protein (Fragment) n=1 Tax=Acinetobacter baumannii 348935 RepID=A0A009DWD5_ACIBA
# Database: uniprot_sprot.fasta
# 0 hits found
# BLASTP 2.4.0+
# Query: UniRef50_A0A009DWJ5 Putative transposase (Fragment) n=1 Tax=Acinetobacter baumannii 348935 RepID=A0A009DWJ5_ACIBA
# Database: uniprot_sprot.fasta
# 0 hits found
# BLASTP 2.4.0+
# Query: UniRef50_A0A009DX03 Uncharacterized protein (Fragment) n=1 Tax=Acinetobacter baumannii 348935 RepID=A0A009DX03_ACIBA
# Database: uniprot_sprot.fasta
# 0 hits found
# BLASTP 2.4.0+
# Query: UniRef50_A0A009DYA3 Putative integrase n=1 Tax=Acinetobacter baumannii 348935 RepID=A0A009DYA3_ACIBA
# Database: uniprot_sprot.fasta
# 0 hits found
# BLASTP 2.4.0+
# Query: UniRef50_A0A009DYI3 Transposase DDE domain protein (Fragment) n=3 Tax=Acinetobacter baumannii 348935 RepID=A0A009DYI3_ACIBA
# Database: uniprot_sprot.fasta
# Fields: query id, subject id, % identity, alignment length, mismatches, gap opens, q. start, q. end, s. start, s. end, evalue, bit score, sbjct frame, query seq, subject seq
# 3 hits found
UniRef50_A0A009DYI3	sp|P55729|Y4ZB_RHISN	39.535	86	52	0	1	86	252	337	3.70e-16	74.7	1	AIYKDRWKVELFFKAIKQNLKLKAFLGRSRNAIQTQIWIAMIAYLLVSFAQHLGKTGWTVQRLLRIIQVNLFERRTLKTLFSPDKI	ALYKGRWQIELLFRWIKQHLKIRSFLGNNDNAVRLQLFAAMIAYALLRIAARLNRITMPILRFTDLVIRCLFERRDIAAIERPPPV
UniRef50_A0A009DYI3	sp|Q55566|T4SA_SYNY3	36.986	73	46	0	2	74	253	325	1.55e-07	50.8	1	IYKDRWKVELFFKAIKQNLKLKAFLGRSRNAIQTQIWIAMIAYLLVSFAQHLGKTGWTVQRLLRIIQVNLFER	IYKKRWQIELLWKFLKMHLKLNRLIAKNENAIGIQIYTCIIAYLILKLLVIPKEAGTTMLDKLRYLQAFMCEK
UniRef50_A0A009DYI3	sp|Q45620|T5377_GEOSE	41.304	46	27	0	2	47	276	321	4.84e-06	46.6	1	IYKDRWKVELFFKAIKQNLKLKAFLGRSRNAIQTQIWIAMIAYLLV	MYQQRWAVEVFFRWVKQYLNVPTLFGTTENAVYNQLFAAFIAYVLL
# BLASTP 2.4.0+
# Query: UniRef50_A0A009DZQ8 Putative membrane protein (Fragment) n=1 Tax=Acinetobacter baumannii 348935 RepID=A0A009DZQ8_ACIBA
# Database: uniprot_sprot.fasta
# 0 hits found
# BLASTP 2.4.0+
# Query: UniRef50_A0A009E0R3 Transposase DDE domain protein (Fragment) n=5 Tax=Acinetobacter RepID=A0A009E0R3_ACIBA
# Database: uniprot_sprot.fasta
# Fields: query id, subject id, % identity, alignment length, mismatches, gap opens, q. start, q. end, s. start, s. end, evalue, bit score, sbjct frame, query seq, subject seq
# 1 hits found
UniRef50_A0A009E0R3	sp|P50358|Y4SN_RHISN	39.241	79	48	0	4	82	28	106	6.58e-15	68.9	1	PCFYCSIFLSIWLYRIRTGCPWRDIPSCFGHSNSIFKRFNRWSSSGKLLRLFKLLASCPDMEWIFIDGSHVRAHQHSAG	PRTNNRLFLDALLWMARSGGRWRDLPERLGDYRAVKRRYYRWIEMGVLDEMLAVLAREADLEWLMIDSTIVRAHQHAAG
# BLASTP 2.4.0+
# Query: UniRef50_A0A009E1D7 Uncharacterized protein (Fragment) n=1 Tax=Acinetobacter baumannii 348935 RepID=A0A009E1D7_ACIBA
# Database: uniprot_sprot.fasta
# 0 hits found
# BLASTP 2.4.0+
# Query: UniRef50_A0A009E284 Uncharacterized protein n=1 Tax=Acinetobacter baumannii 348935 RepID=A0A009E284_ACIBA
# Database: uniprot_sprot.fasta
# 0 hits found
# BLASTP 2.4.0+
# Query: UniRef50_A0A009E2R5 Toxin-antitoxin system, antitoxin component, Xre family n=1 Tax=Acinetobacter baumannii 348935 RepID=A0A009E2R5_ACIBA
# Database: uniprot_sprot.fasta
# 0 hits found
# BLASTP 2.4.0+
# Query: UniRef50_A0A009E3M2 Glycosyl transferases group 1 family protein (Fragment) n=1 Tax=Acinetobacter baumannii 348935 RepID=A0A009E3M2_ACIBA
# Database: uniprot_sprot.fasta
# Fields: query id, subject id, % identity, alignment length, mismatches, gap opens, q. start, q. end, s. start, s. end, evalue, bit score, sbjct frame, query seq, subject seq
# 8 hits found
UniRef50_A0A009E3M2	sp|Q0P9C7|PGLJ_CAMJE	32.044	181	119	2	98	274	152	332	1.96e-20	93.6	1	SQGVEDDLKETLGDLPN-VSTIYNAFDQHYMQNQANEFQPELGKYLLHVGAFCYAKAHDTLIKAYAESSQQIPLYLLGKGELEADIKQLIYSLNLEKKVHFLGFNKNPYPFIKNAQALILSSRYEGFVRVISEALALGTPVISTNCPSGPNEILSPNN---LVAVDDIKALAEKMNQVMQN	SKGNLEDLVQNFSISPKKCEILYNAIDLENIGQKALEDIALKDKFILSVGRLDKGKNHALLIRAYARLKTDLKLVILGEGVLKDELLALIKELNLEEKVLLLGFDNNPYKYMAKCEFFAFASVFEGFSNVLIESLACSCAVVCTDHKSGARELFGDDEFGLLVEVDNENSMFQGLKTMLED
UniRef50_A0A009E3M2	sp|Q58459|Y1059_METJA	38.596	114	67	1	142	252	205	318	2.28e-20	93.6	1	LLHVGAFCYAKAHDTLIKAYAESSQQIP---LYLLGKGELEADIKQLIYSLNLEKKVHFLGFNKNPYPFIKNAQALILSSRYEGFVRVISEALALGTPVISTNCPSGPNEILSP	FINIGRLTEQKGQWFLIRSFKRVTEKYPNAKLIILGDGELKNKLQELINKLNLQNNVYLLGMQKNPFKFLKHSNCFVFSSLWEGLPNTVIEALSLNLPVISTDCKTGPREILCP
UniRef50_A0A009E3M2	sp|Q9R9N1|LPSE_RHIME	26.154	195	129	7	89	274	116	304	5.08e-12	68.9	1	YKKHPCI-CVSQGVEDDLKETLGDLPNVSTIYN--AFDQHYMQNQANEFQPELGKYLLHVGAFCYAKAHDTLIKAYAESSQQIP---LYLLGKGELEADIKQLIYSLNLEKKVHFLGFNKNPYPFIKNAQALILSSRYEGFVRVISEALALGTPVISTNCPSGPNEILSPNN---LVAVDDIKALAEKMNQVMQN	FRNTDCIVCNTPGIAERVSD-LGWKREIRVISNFTGTGRVVAVDRAKLDTPADAPVVMSMGRFVERKGFHTLIEAVA----RLPGVYLWLLGDGEERDNLHKLATDLGVSGRVRFAGWQDDTRPFLAAVDVFVMSSSHEPLGNVILESWAQGTPVVSTRS-EGPQWFMRDGENGLMVDIGDAEGFARAIEQIVAD
UniRef50_A0A009E3M2	sp|Q81ST7|BSHA_BACAN	25.694	288	163	11	1	274	87	337	2.94e-11	67.0	1	IDIKHWYYKLFIIITWRYQYIAKQFDQYVLKNIGQPDLILSNLIQTNRIL--SNSKLDNIAYVIRNTFSKENENALLKNPNKVINRYREIYKKHPCICVSQGVEDDLKETLGDLPN--VSTIYNAFDQH-YMQNQANEFQPELG-----KYLLHVGAFCYAKAHDTLIKAYAESSQQI--PLYLLGKGELEADIKQLIYSLNLEKKVHFLGFNKNPYPFIKNAQALILSSRYEGFVRVISEALALGTPVISTNCPSGPNEILSPNN--LVAVDDIKALAEKMNQVMQN	LDILHVHYAIPHAICA---YLAKQM-------IGERIKIVTTLHGTDITVLGSDPSLNNL---IR--FGIEQSDVVTAVSHSLINETHELVK----------------------PNKDIQTVYNFIDERVYFKRDMTQLKKEYGISESEKILIHISNFRKVKRVQDVVQAFAKIVTEVDAKLLLVGDGPEFCTILQLVKNLHIEDRVLFLGKQDNVAELLAMSDLMLLLSEKESFGLVLLEAMACGVPCIGTRVGGIPEVIQHGDTGYLCEVGDTTGVADQAIQLLKD
UniRef50_A0A009E3M2	sp|Q06994|RFAB_SALTY	24.675	231	148	9	89	300	124	347	2.12e-10	64.3	1	YKKHP-CI-------CVSQGVEDDLKETLGDLPNVSTIYNAFD-QHYMQNQANEFQPELGKYLLHVG--AFCYAKAHDTLIKAYAESSQQIPLYLLGKGELEADIKQLIYSLNLEKKVHFLGFNKNPYPF----IKNAQALILSSRYEGFVRVISEALALGTPVISTNCPSGPNEILSP---NNLVAVDDIKALAEKMNQVMQNPNDFYTPFNLNFLPCNIAKQY-LDYFK	HKKHAECITYADYHLAISSGIKEQIMARGISAQDISVVYNPVSIKTVIVPPPERDKPAV---FLYVGRLKFEGQKRVKDLFDGLARTTGEWQLHIIGDGSDFEKCQAYSRELGIEQRVIWYGWQSAPWQVVQQKIKNVTALLLTSAFEGFPMTLLEAMSYGIPCISSDCMSGPRDMIKPGLNGELYTPGAIDDFVGHLNRVISGE----VKYQHDIIPGTIERFYDVLYFK
UniRef50_A0A009E3M2	sp|P27127|RFAB_ECOLI	28.099	121	80	2	157	270	198	318	2.33e-09	61.2	1	LIKAYAESSQQIPLYLLGKGELEADIKQLIYSLNLEKKVHFLGFNKNPYPF----IKNAQALILSSRYEGFVRVISEALALGTPVISTNCPSGPNEILSPN---NLVAVDDIKALAEKMNQ	LLDGLSQAKGNWKLHVLGDGSDFEKCQAYGRELNIDDRIVWYGWQQYPWELVQQDIEKVSALLLTSSFEGFPMTLLEALSWGIPCISADCVSGPADIIQPDVNGHLYQPGDIAGFVTLLNK
UniRef50_A0A009E3M2	sp|O05083|Y1698_HAEIN	26.056	142	99	2	140	275	182	323	8.24e-09	59.3	1	KYLLHVGAFCYAKAHDTLIKAYAESSQQIP---LYLLGKGELEADIKQLIYSLNLEKKVHFLGFNKNPYPFIKNAQALILSSRYEGFVRVISEALALGTPVISTNCPSGPNEILSPNN---LVAVDDIKALAEKMNQVMQNP	KTILSVGHLFSYKGFDYLLKVWQVLAKKYPDWNLKIVGSGEEEENLKNLAKALDIEDSVNFIPRTNDVSFYYESSSIYCLPSQTEGLPLVVIEAMAFGLPIVAFNCSPGVKQLVEHKENGFLCEQNNIEEMVKGLDLLINNP
UniRef50_A0A009E3M2	sp|Q9R9N0|LPSD_RHIME	24.419	172	118	4	142	303	171	340	1.30e-08	58.9	1	LLHVGAFCYAKAHDTLIKAYAESSQQIPLYLLGKGELEADIKQLIYSLNLEKKVHFLGFNKNPYPFIKNAQALILSSRYEGFVRVISEALALGTPVISTNCPSGPNEILSPNN---LVAVDDIKALAEKMNQVMQNP-------NDFYTPFNLNFLPCNIAKQYLDYFKINI	VVGMGRFVKRKGFAGLIRAVKEV-ENTYLWLVGDGPEREQLEQLTDELGLRDRVRFTGWQTNAYGFLSAGDAFVINSSHEPLGNVCFEGWGAGKPTIASRA-EGPSWVMTHESDALMVDCGDDVGLAAAIRRLRDDPALRERLSAGGSETLRTRFSEKAITDAYLDLFDRGV
