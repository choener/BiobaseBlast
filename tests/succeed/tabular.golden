[BlastTabularResult {blastProgram = BlastP, blastQueryId = "UniRef50_A0A009DWD5", blastDatabase = "uniprot_sprot.fasta", blastHitNumber = 0, hitLines = []},BlastTabularResult {blastProgram = BlastP, blastQueryId = "UniRef50_A0A009DWJ5", blastDatabase = "uniprot_sprot.fasta", blastHitNumber = 0, hitLines = []},BlastTabularResult {blastProgram = BlastP, blastQueryId = "UniRef50_A0A009DX03", blastDatabase = "uniprot_sprot.fasta", blastHitNumber = 0, hitLines = []},BlastTabularResult {blastProgram = BlastP, blastQueryId = "UniRef50_A0A009DYA3", blastDatabase = "uniprot_sprot.fasta", blastHitNumber = 0, hitLines = []},BlastTabularResult {blastProgram = BlastP, blastQueryId = "UniRef50_A0A009DYI3", blastDatabase = "uniprot_sprot.fasta", blastHitNumber = 3, hitLines = [BlastTabularHit {queryId = "UniRef50_A0A009DYI3", subjectId = "sp|P55729|Y4ZB_RHISN", seqIdentity = 39.535, alignmentLength = 86, misMatches = 52, gapOpenScore = 0, queryStart = 1, queryEnd = 86, hitSeqStart = 252, hitSeqEnd = 337, eValue = 3.7e-16, bitScore = 74.7, subjectFrame = 1, querySeq = "AIYKDRWKVELFFKAIKQNLKLKAFLGRSRNAIQTQIWIAMIAYLLVSFAQHLGKTGWTVQRLLRIIQVNLFERRTLKTLFSPDKI", subjectSeq = "ALYKGRWQIELLFRWIKQHLKIRSFLGNNDNAVRLQLFAAMIAYALLRIAARLNRITMPILRFTDLVIRCLFERRDIAAIERPPPV"},BlastTabularHit {queryId = "UniRef50_A0A009DYI3", subjectId = "sp|Q55566|T4SA_SYNY3", seqIdentity = 36.986, alignmentLength = 73, misMatches = 46, gapOpenScore = 0, queryStart = 2, queryEnd = 74, hitSeqStart = 253, hitSeqEnd = 325, eValue = 1.55e-7, bitScore = 50.8, subjectFrame = 1, querySeq = "IYKDRWKVELFFKAIKQNLKLKAFLGRSRNAIQTQIWIAMIAYLLVSFAQHLGKTGWTVQRLLRIIQVNLFER", subjectSeq = "IYKKRWQIELLWKFLKMHLKLNRLIAKNENAIGIQIYTCIIAYLILKLLVIPKEAGTTMLDKLRYLQAFMCEK"},BlastTabularHit {queryId = "UniRef50_A0A009DYI3", subjectId = "sp|Q45620|T5377_GEOSE", seqIdentity = 41.304, alignmentLength = 46, misMatches = 27, gapOpenScore = 0, queryStart = 2, queryEnd = 47, hitSeqStart = 276, hitSeqEnd = 321, eValue = 4.84e-6, bitScore = 46.6, subjectFrame = 1, querySeq = "IYKDRWKVELFFKAIKQNLKLKAFLGRSRNAIQTQIWIAMIAYLLV", subjectSeq = "MYQQRWAVEVFFRWVKQYLNVPTLFGTTENAVYNQLFAAFIAYVLL"}]},BlastTabularResult {blastProgram = BlastP, blastQueryId = "UniRef50_A0A009DZQ8", blastDatabase = "uniprot_sprot.fasta", blastHitNumber = 0, hitLines = []},BlastTabularResult {blastProgram = BlastP, blastQueryId = "UniRef50_A0A009E0R3", blastDatabase = "uniprot_sprot.fasta", blastHitNumber = 1, hitLines = [BlastTabularHit {queryId = "UniRef50_A0A009E0R3", subjectId = "sp|P50358|Y4SN_RHISN", seqIdentity = 39.241, alignmentLength = 79, misMatches = 48, gapOpenScore = 0, queryStart = 4, queryEnd = 82, hitSeqStart = 28, hitSeqEnd = 106, eValue = 6.58e-15, bitScore = 68.9, subjectFrame = 1, querySeq = "PCFYCSIFLSIWLYRIRTGCPWRDIPSCFGHSNSIFKRFNRWSSSGKLLRLFKLLASCPDMEWIFIDGSHVRAHQHSAG", subjectSeq = "PRTNNRLFLDALLWMARSGGRWRDLPERLGDYRAVKRRYYRWIEMGVLDEMLAVLAREADLEWLMIDSTIVRAHQHAAG"}]},BlastTabularResult {blastProgram = BlastP, blastQueryId = "UniRef50_A0A009E1D7", blastDatabase = "uniprot_sprot.fasta", blastHitNumber = 0, hitLines = []},BlastTabularResult {blastProgram = BlastP, blastQueryId = "UniRef50_A0A009E284", blastDatabase = "uniprot_sprot.fasta", blastHitNumber = 0, hitLines = []},BlastTabularResult {blastProgram = BlastP, blastQueryId = "UniRef50_A0A009E2R5", blastDatabase = "uniprot_sprot.fasta", blastHitNumber = 0, hitLines = []},BlastTabularResult {blastProgram = BlastP, blastQueryId = "UniRef50_A0A009E3M2", blastDatabase = "uniprot_sprot.fasta", blastHitNumber = 8, hitLines = [BlastTabularHit {queryId = "UniRef50_A0A009E3M2", subjectId = "sp|Q0P9C7|PGLJ_CAMJE", seqIdentity = 32.044, alignmentLength = 181, misMatches = 119, gapOpenScore = 2, queryStart = 98, queryEnd = 274, hitSeqStart = 152, hitSeqEnd = 332, eValue = 1.96e-20, bitScore = 93.6, subjectFrame = 1, querySeq = "SQGVEDDLKETLGDLPN-VSTIYNAFDQHYMQNQANEFQPELGKYLLHVGAFCYAKAHDTLIKAYAESSQQIPLYLLGKGELEADIKQLIYSLNLEKKVHFLGFNKNPYPFIKNAQALILSSRYEGFVRVISEALALGTPVISTNCPSGPNEILSPNN---LVAVDDIKALAEKMNQVMQN", subjectSeq = "SKGNLEDLVQNFSISPKKCEILYNAIDLENIGQKALEDIALKDKFILSVGRLDKGKNHALLIRAYARLKTDLKLVILGEGVLKDELLALIKELNLEEKVLLLGFDNNPYKYMAKCEFFAFASVFEGFSNVLIESLACSCAVVCTDHKSGARELFGDDEFGLLVEVDNENSMFQGLKTMLED"},BlastTabularHit {queryId = "UniRef50_A0A009E3M2", subjectId = "sp|Q58459|Y1059_METJA", seqIdentity = 38.596, alignmentLength = 114, misMatches = 67, gapOpenScore = 1, queryStart = 142, queryEnd = 252, hitSeqStart = 205, hitSeqEnd = 318, eValue = 2.28e-20, bitScore = 93.6, subjectFrame = 1, querySeq = "LLHVGAFCYAKAHDTLIKAYAESSQQIP---LYLLGKGELEADIKQLIYSLNLEKKVHFLGFNKNPYPFIKNAQALILSSRYEGFVRVISEALALGTPVISTNCPSGPNEILSP", subjectSeq = "FINIGRLTEQKGQWFLIRSFKRVTEKYPNAKLIILGDGELKNKLQELINKLNLQNNVYLLGMQKNPFKFLKHSNCFVFSSLWEGLPNTVIEALSLNLPVISTDCKTGPREILCP"},BlastTabularHit {queryId = "UniRef50_A0A009E3M2", subjectId = "sp|Q9R9N1|LPSE_RHIME", seqIdentity = 26.154, alignmentLength = 195, misMatches = 129, gapOpenScore = 7, queryStart = 89, queryEnd = 274, hitSeqStart = 116, hitSeqEnd = 304, eValue = 5.08e-12, bitScore = 68.9, subjectFrame = 1, querySeq = "YKKHPCI-CVSQGVEDDLKETLGDLPNVSTIYN--AFDQHYMQNQANEFQPELGKYLLHVGAFCYAKAHDTLIKAYAESSQQIP---LYLLGKGELEADIKQLIYSLNLEKKVHFLGFNKNPYPFIKNAQALILSSRYEGFVRVISEALALGTPVISTNCPSGPNEILSPNN---LVAVDDIKALAEKMNQVMQN", subjectSeq = "FRNTDCIVCNTPGIAERVSD-LGWKREIRVISNFTGTGRVVAVDRAKLDTPADAPVVMSMGRFVERKGFHTLIEAVA----RLPGVYLWLLGDGEERDNLHKLATDLGVSGRVRFAGWQDDTRPFLAAVDVFVMSSSHEPLGNVILESWAQGTPVVSTRS-EGPQWFMRDGENGLMVDIGDAEGFARAIEQIVAD"},BlastTabularHit {queryId = "UniRef50_A0A009E3M2", subjectId = "sp|Q81ST7|BSHA_BACAN", seqIdentity = 25.694, alignmentLength = 288, misMatches = 163, gapOpenScore = 11, queryStart = 1, queryEnd = 274, hitSeqStart = 87, hitSeqEnd = 337, eValue = 2.94e-11, bitScore = 67.0, subjectFrame = 1, querySeq = "IDIKHWYYKLFIIITWRYQYIAKQFDQYVLKNIGQPDLILSNLIQTNRIL--SNSKLDNIAYVIRNTFSKENENALLKNPNKVINRYREIYKKHPCICVSQGVEDDLKETLGDLPN--VSTIYNAFDQH-YMQNQANEFQPELG-----KYLLHVGAFCYAKAHDTLIKAYAESSQQI--PLYLLGKGELEADIKQLIYSLNLEKKVHFLGFNKNPYPFIKNAQALILSSRYEGFVRVISEALALGTPVISTNCPSGPNEILSPNN--LVAVDDIKALAEKMNQVMQN", subjectSeq = "LDILHVHYAIPHAICA---YLAKQM-------IGERIKIVTTLHGTDITVLGSDPSLNNL---IR--FGIEQSDVVTAVSHSLINETHELVK----------------------PNKDIQTVYNFIDERVYFKRDMTQLKKEYGISESEKILIHISNFRKVKRVQDVVQAFAKIVTEVDAKLLLVGDGPEFCTILQLVKNLHIEDRVLFLGKQDNVAELLAMSDLMLLLSEKESFGLVLLEAMACGVPCIGTRVGGIPEVIQHGDTGYLCEVGDTTGVADQAIQLLKD"},BlastTabularHit {queryId = "UniRef50_A0A009E3M2", subjectId = "sp|Q06994|RFAB_SALTY", seqIdentity = 24.675, alignmentLength = 231, misMatches = 148, gapOpenScore = 9, queryStart = 89, queryEnd = 300, hitSeqStart = 124, hitSeqEnd = 347, eValue = 2.12e-10, bitScore = 64.3, subjectFrame = 1, querySeq = "YKKHP-CI-------CVSQGVEDDLKETLGDLPNVSTIYNAFD-QHYMQNQANEFQPELGKYLLHVG--AFCYAKAHDTLIKAYAESSQQIPLYLLGKGELEADIKQLIYSLNLEKKVHFLGFNKNPYPF----IKNAQALILSSRYEGFVRVISEALALGTPVISTNCPSGPNEILSP---NNLVAVDDIKALAEKMNQVMQNPNDFYTPFNLNFLPCNIAKQY-LDYFK", subjectSeq = "HKKHAECITYADYHLAISSGIKEQIMARGISAQDISVVYNPVSIKTVIVPPPERDKPAV---FLYVGRLKFEGQKRVKDLFDGLARTTGEWQLHIIGDGSDFEKCQAYSRELGIEQRVIWYGWQSAPWQVVQQKIKNVTALLLTSAFEGFPMTLLEAMSYGIPCISSDCMSGPRDMIKPGLNGELYTPGAIDDFVGHLNRVISGE----VKYQHDIIPGTIERFYDVLYFK"},BlastTabularHit {queryId = "UniRef50_A0A009E3M2", subjectId = "sp|P27127|RFAB_ECOLI", seqIdentity = 28.099, alignmentLength = 121, misMatches = 80, gapOpenScore = 2, queryStart = 157, queryEnd = 270, hitSeqStart = 198, hitSeqEnd = 318, eValue = 2.33e-9, bitScore = 61.2, subjectFrame = 1, querySeq = "LIKAYAESSQQIPLYLLGKGELEADIKQLIYSLNLEKKVHFLGFNKNPYPF----IKNAQALILSSRYEGFVRVISEALALGTPVISTNCPSGPNEILSPN---NLVAVDDIKALAEKMNQ", subjectSeq = "LLDGLSQAKGNWKLHVLGDGSDFEKCQAYGRELNIDDRIVWYGWQQYPWELVQQDIEKVSALLLTSSFEGFPMTLLEALSWGIPCISADCVSGPADIIQPDVNGHLYQPGDIAGFVTLLNK"},BlastTabularHit {queryId = "UniRef50_A0A009E3M2", subjectId = "sp|O05083|Y1698_HAEIN", seqIdentity = 26.056, alignmentLength = 142, misMatches = 99, gapOpenScore = 2, queryStart = 140, queryEnd = 275, hitSeqStart = 182, hitSeqEnd = 323, eValue = 8.24e-9, bitScore = 59.3, subjectFrame = 1, querySeq = "KYLLHVGAFCYAKAHDTLIKAYAESSQQIP---LYLLGKGELEADIKQLIYSLNLEKKVHFLGFNKNPYPFIKNAQALILSSRYEGFVRVISEALALGTPVISTNCPSGPNEILSPNN---LVAVDDIKALAEKMNQVMQNP", subjectSeq = "KTILSVGHLFSYKGFDYLLKVWQVLAKKYPDWNLKIVGSGEEEENLKNLAKALDIEDSVNFIPRTNDVSFYYESSSIYCLPSQTEGLPLVVIEAMAFGLPIVAFNCSPGVKQLVEHKENGFLCEQNNIEEMVKGLDLLINNP"},BlastTabularHit {queryId = "UniRef50_A0A009E3M2", subjectId = "sp|Q9R9N0|LPSD_RHIME", seqIdentity = 24.419, alignmentLength = 172, misMatches = 118, gapOpenScore = 4, queryStart = 142, queryEnd = 303, hitSeqStart = 171, hitSeqEnd = 340, eValue = 1.3e-8, bitScore = 58.9, subjectFrame = 1, querySeq = "LLHVGAFCYAKAHDTLIKAYAESSQQIPLYLLGKGELEADIKQLIYSLNLEKKVHFLGFNKNPYPFIKNAQALILSSRYEGFVRVISEALALGTPVISTNCPSGPNEILSPNN---LVAVDDIKALAEKMNQVMQNP-------NDFYTPFNLNFLPCNIAKQYLDYFKINI", subjectSeq = "VVGMGRFVKRKGFAGLIRAVKEV-ENTYLWLVGDGPEREQLEQLTDELGLRDRVRFTGWQTNAYGFLSAGDAFVINSSHEPLGNVCFEGWGAGKPTIASRA-EGPSWVMTHESDALMVDCGDDVGLAAAIRRLRDDPALRERLSAGGSETLRTRFSEKAITDAYLDLFDRGV"}]}]
